src/main/osd/osd.h

```diff
@@ -186,6 +186,9 @@ typedef enum {
     OSD_SYS_WARNINGS,
     OSD_SYS_VTX_TEMP,
     OSD_SYS_FAN_SPEED,
+    OSD_GPS_LAP_TIME_CURRENT,
```

Comment:
Any value greater than or equal to OSD_SYS_GOGGLE_VOLTAGE is treated as a system element and won't be rendered. I suggest the following fix. 

```
diff --git a/src/main/osd/osd_elements.c b/src/main/osd/osd_elements.c
index ecc72f9..ea82f15 100644
--- a/src/main/osd/osd_elements.c
+++ b/src/main/osd/osd_elements.c
@@ -2016,7 +2016,7 @@ static void osdDrawSingleElement(displayPort_t *osdDisplayPort, uint8_t item)
     element.attr = DISPLAYPORT_SEVERITY_NORMAL;
 
     // Call the element drawing function
-    if ((item >= OSD_SYS_GOGGLE_VOLTAGE) && (item < OSD_ITEM_COUNT)) {
+    if ((item >= OSD_SYS_GOGGLE_VOLTAGE) && (item < OSD_GPS_LAP_TIME_CURRENT)) {
         displaySys(osdDisplayPort, elemPosX, elemPosY, (displayPortSystemElement_e)(item - OSD_SYS_GOGGLE_VOLTAGE + DISPLAYPORT_SYS_GOGGLE_VOLTAGE));
     } else {
         osdElementDrawFunction[item](&element);
```

Reply:
With the above change...
![image](https://github.com/betaflight/betaflight/assets/11480839/b7b32671-ae7b-4067-a606-675e7b00cf27)